web-components/src/components/icons/ToucanIcon.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { SxProps, Theme } from '@mui/material';
+import SvgIcon from '@mui/material/SvgIcon';
+
+interface ToucanIconProps {
+  sx?: SxProps<Theme>;
+}
+
+export function ToucanIcon({ sx }: ToucanIconProps): JSX.Element {
+  return (
+    <SvgIcon sx={sx} viewBox="0 0 109 32">
+      <svg
```

Comment:
I think this SVG is based on an old one, in the more recent ones we try to:
- Extends SvgIconProps for Props
- Avoid duplicate `svg` tags
- Use `currentColor` for fill to make it customizable

A more up-to-date example would [this one](https://github.com/regen-network/regen-web/blob/master/web-components/src/components/icons/EmptyCartIcon.tsx) for example.

Reply:
updated. But I didn't bother with `currentColor` since this is likely to only be used in full-color AFAIK.